app/containers/Products/ProductCreatorContainer.tsx

```diff
@@ -76,19 +85,20 @@ export const ProductCreator: React.FunctionComponent<Props> = ({
               onSubmit={saveProduct}
             >
               <Button type="submit">Add Product</Button>
-              <Button variant="danger" onClick={toggleShowCreateForm}>
+              <Button
+                variant="secondary"
+                style={{marginLeft: '0.5em'}}
+                onClick={toggleShowCreateForm}
+              >
                 Close
               </Button>
             </JsonSchemaForm>
           </Card.Body>
         </Card>
       </div>
-      <style jsx>{`
-        .hidden-title label {
-          display: none;
-        }
-        div {
-          margin-bottom: 50px;
+      <style jsx global>{`
```

Comment:
```
      <style jsx global>
```

This was necessary to punch through into the form's help text style to give it a darker color. 

That was required because the darker (grey) form background made the contrast with the help text insufficient for AA contrast.

Reply:
Given that the CIIP application forms should have similar changes applied to them for consistency, I would suggest changing the class of the help button instead:
https://github.com/bcgov/cas-ciip-portal/blob/8690490de0a5fbde96649d34c57cec28c451fb39/app/containers/Forms/FormFieldTemplate.tsx#L30